workers/executor/executors/legacy_executor.py

```
@@ -1664,6 +1738,82 @@ def _run_table_extraction(
+        from executor.executors.constants import PromptServiceConstants as PSKeys
+
+        prompt_name = prompt_run_args["prompt_name"]
+        try:
+            line_item_executor = ExecutorRegistry.get("line_item")
+        except KeyError:
+            raise LegacyExecutorError(
+                message=(
+                    "LINE_ITEM extraction requires the line_item executor "
+                    "plugin. Install the line_item_extractor plugin."
+                )
+            )
```

Ruff (0.15.9)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)